build-support/bin/node/mac/10.15/v12.18.2/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -exu
+
+# Build Node.js for just this version and architecture.
```

Comment:
Could you please symlink the older OSX directories to this one [as in the readme](https://github.com/pantsbuild/binaries#workflow)? Thanks!!